integration_tests/integration/bookedtoday/arrivals/checkAnswers.spec.ts

```diff
@@ -23,50 +32,92 @@ context('Check Answers', () => {
   it('Should contain a full set of correctly formatted move data', () => {
     cy.task('stubExpectedArrival', expectedArrival)
     cy.signIn()
-    const existingRecordPage = ExistingRecordPage.goTo(expectedArrival.id)
-    existingRecordPage.continue().click()
+
+    const singleRecordFoundPage = ChoosePrisonerPage.selectPrisoner(expectedArrival.id, SingleRecordFoundPage)
+    singleRecordFoundPage.perName().should('contain.text', 'Bob Smith')
+    singleRecordFoundPage.perDob().should('contain.text', '1 January 1970')
+    singleRecordFoundPage.perPrisonNumber().should('contain.text', 'G0015GF')
+    singleRecordFoundPage.perPncNumber().should('contain.text', '01/2345A')
+    singleRecordFoundPage.continue().should('have.attr', 'href', `/prisoners/${expectedArrival.id}/sex`)
+
+    singleRecordFoundPage.existingName().should('contain.text', 'Sam Smith')
+    singleRecordFoundPage.existingDob().should('contain.text', '1 February 1970')
+    singleRecordFoundPage.existingPrisonNumber().should('contain.text', 'A1234BC')
+    singleRecordFoundPage.existingPncNumber().should('contain.text', '01/4567A')
+    singleRecordFoundPage.continue().click()
 
     const sexPage = Page.verifyOnPage(SexPage)
     sexPage.sexRadioButtons('F').click()
     sexPage.continue().click()
 
     const imprisonmentStatusPage = ImprisonmentStatusPage.goTo(expectedArrival.id)
-    imprisonmentStatusPage.imprisonmentStatusRadioButton('on-remand').click()
     imprisonmentStatusPage.continue().click()
+    imprisonmentStatusPage.hasError('Select a reason for imprisonment')
+    imprisonmentStatusPage.imprisonmentStatusRadioButton('determinate-sentence').click()
+    imprisonmentStatusPage.continue().click()
+
+    const movementReasonPage = Page.verifyOnPage(MovementReasonsPage)
+    movementReasonPage.continue().click()
+    movementReasonPage.hasError('Select the type of determinate sentence')
+    movementReasonPage.movementReasonRadioButton('26').click()
+    movementReasonPage.continue().click()
 
-    const checkAnswersPage = CheckAnswersPage.goTo(expectedArrival.id)
```

Comment:
Existing, but we shouldn't really be using `goTo` anywhere apart from the entry of a test